pkgs/development/python-modules/aws-adfs/default.nix

```
@@ -1,13 +1,14 @@
 { lib, buildPythonPackage, fetchPypi
-, pytest, pytestrunner, pytestcov, mock, glibcLocales, lxml, boto3, requests, click, configparser }:
+, pytest, pytestrunner, pytestcov, mock, glibcLocales, lxml, boto3
+, requests, click, configparser, fido2 }:
 
 buildPythonPackage rec {
   pname = "aws-adfs";
-  version = "1.19.1";
+  version = "1.20.0";
```

to "fix" python2 issue
```suggestion
  version = "1.20.0";
  disabled = isPy27;
```